src/dms/core/EdFi.DataManagementService.Core/Security/AuthorizationFilters/RelationshipsWithEdOrgsAndPeopleFiltersProvider.cs

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Core.External.Backend;
+using EdFi.DataManagementService.Core.External.Model;
+
+namespace EdFi.DataManagementService.Core.Security.AuthorizationFilters;
+
+/// <summary>
+/// Provides authorization filters for RelationshipsWithEdOrgsOnly authorization strategy
+/// </summary>
+[AuthorizationStrategyName(AuthorizationStrategyName)]
+public class RelationshipsWithEdOrgsAndPeopleFiltersProvider : IAuthorizationFiltersProvider
```

Comment:
This will temporarily mimic the implementation of RelationshipsWithEdOrgsOnlyFiltersProvider. It will be refactored as part of the SSA "get by query" implementation.